tests/contrib/test_torch.py

```diff
@@ -174,6 +177,25 @@ def test_torch_import(tmp_path, net, torchsave):
     assert isinstance(meta.model_type, TorchModel)
 
 
+def test_torch_import_in_separate_shell():
+    tmp_path = mkdtemp()
```

Comment:
use fixture instead

Reply:
We want a temp dir that persists between different shells / terminals.
If I use the existing fixture, that doesn't persist.

So we need to use `mkdtemp ` which is not automatically deleted.
I can make a new fixture that uses `mkdtemp`, does that work?

Reply:
WDYM `persists between different shells`?

Reply:
it's just a path. and you generate and unlink it in the same test instance

Reply:
By persist, I mean that, when I run the subprocess call, that new shell / terminal should be able to access the tempdir.

By default, dirs created by the `tempfile` library delete themselves automatically unless one uses `mkdtemp`

Reply:
done